docs/general/dql/selects.rst

```diff
@@ -740,6 +740,43 @@ an upper-bound::
     Both the ``from`` and ``to`` index values are inclusive.
     Using an index greater than the array size results in an empty array.
 
+For selecting records having values with empty or unassigned arrays, like::
+
+    cr> insert into locations (id, name, position, kind, landmarks) values
+    ... (15, 'Voidspace', 0, 'Milliways', []);
+    INSERT OK, 1 row affected (... sec)
+
+.. Hidden: refresh locations
+
+    cr> refresh table locations;
+    REFRESH OK, 1 row affected (... sec)
+
+a suitable expression to match for both ``NULL`` values and empty arrays, is::
+
+    cr> select count(*) from locations where
+    ... landmarks is null or
+    ... landmarks = [];
```

Comment:
As written here #9398#issuecomment-559750730, `IS NULL` will also match empty arrays, so I think there is no need for additional documentation.

Reply:
Ack. I will amend this patch to solely use `IS NULL` as a check for both emptyness or nullness.